authentication/serializers.py

```diff
@@ -49,7 +49,7 @@ def validate(self, attrs):
                 raise serializers.ValidationError(
                     "Please login using @itbhu.ac.in student email id only")
             name = jwt['name']
-            user = User()
+            user = get_user_model()
```

Comment:
@nb9960 Should this be `user = get_user_model()()` instead? I think this method returns a user model and not the user object.

Something like this should work imo:

```py
UserModel = get_user_model()
user = UserModel()  # or user = UserModel.objects.create_user(...)
...
user.save()
```

cc: @Vikhyath08 
https://github.com/IIT-BHU-InstiApp/IIT-BHU-app/issues/339: Seems like this is a general issue, not related to branch changers.

I fixed a similar bug last time by just reverting the change :stuck_out_tongue: (https://github.com/IIT-BHU-InstiApp/lite-hai-backend/pull/76), but directly using `User` might not be good if someone is using custom user model.

Reply:
Great! :tada: